### PR TITLE
[Northumberland] Add contact-by-phone information to FAQ

### PR DESCRIPTION
**Pull request by chrismytton**
Add a section to the FAQ page about contacting the council by phone.

Fixes https://github.com/mysociety/societyworks/issues/3629

To see the actual changes you can run:

```
git diff --no-index templates/web/fixmystreet-uk-councils/about/faq-en-gb.html templates/web/northumberland/about/faq-en-gb.html
```

### CODE REVIEW

File: templates/web/northumberland/about/faq-en-gb.html [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    For these types of issue, please
+    [%~ IF c.cobrand.feature('contact_us_url') %]
+     <a href="[% c.cobrand.feature('contact_us_url') %]">contact us</a>
+    [%~ ELSE %]
+    contact us
+    [%~ END %]
+    through the appropriate channel.
+</p>
+
+<b>Contact us by phone</b>
```
Comment (dracos) [inline templates/web/northumberland/about/faq-en-gb.html:39]: This probably should be wrapped in a `<p>`

File: templates/web/northumberland/about/faq-en-gb.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+<ul>
+    <li>Monday to Thursday 08:30 - 17:00</li>
+    <li>Friday 08:30 - 16:30</li>
+</ul>
+
+<p><b>PLEASE NOTE</b>: Calls outside of the hours above are for <b>emergency cases only</b> and will be answered by Northumberland Fire & Rescue Service.</p>
+
+<b>Customers with speed or hearing difficulties</b>
+
+<p>If you have speech of hearing difficulties you acn use Relay UK to contact us by dialing 018001 01670 623515. <a href="https://www.relayuk.bt.com/">Find more information about Relay UK</a>.</p>
```
Comment (dracos) [inline templates/web/northumberland/about/faq-en-gb.html:52]: Couple of typos here:
```suggestion
<p>If you have speech or hearing difficulties you can use Relay UK to contact us by dialling 018001 01670 623515. <a href="https://www.relayuk.bt.com/">Find more information about Relay UK</a>.</p>
```

File: templates/web/northumberland/about/faq-en-gb.html [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+<p>You can call us on 0345 600 6400:</p>
+
+<ul>
+    <li>Monday to Thursday 08:30 - 17:00</li>
+    <li>Friday 08:30 - 16:30</li>
+</ul>
+
+<p><b>PLEASE NOTE</b>: Calls outside of the hours above are for <b>emergency cases only</b> and will be answered by Northumberland Fire & Rescue Service.</p>
+
+<b>Customers with speed or hearing difficulties</b>
```
Comment (dracos) [inline templates/web/northumberland/about/faq-en-gb.html:50]: And here (also within a `<p>`):
```suggestion
<b>Customers with speech or hearing difficulties</b>
```

**Review by dracos — CHANGES_REQUESTED** [review]
Some typos I'm afraid

**Comment by chrismytton** [comment]
That'll teach me to copy/paste without proof reading 🙄

**Review by dracos — APPROVED** [review]
☎️

**Pull request merged**